themes/events/layouts/shortcodes/eventlist.html

```diff
@@ -1,13 +1,18 @@
 {{- $type := .Get "type" -}} <!-- options: year, range -->
 {{- $date := .Get "date" -}} <!-- options: all, YYYY, next_12m, prev_12m, next_all, prev_all -->
 {{- $month_counter := "0" -}} <!-- set to non-existing month number to let month name display initially -->
+{{- $homepage := .Get "homepage" -}}
```

Comment:
Is it somehow possible to just pass every event field?

Reply:
Yes, see line 64 where we pass the context `.` as variable `context` to event.html. That means, you can access the variable homepage in the md file as `.context.Params.homepage` as we do it in L8 of event.html. Thereby, `cfp` would be accessible with `.context.Params.cfp`

Reply:
The reason why we have to do it this way is because we also have to pass variables generated in eventlist.html to event.html, e.g. $day

Reply:
Again for clarification: defining $homepage etc by L4-L7 in this PR is not necessary. We only run .Get for the parameters we pass to the shortcode {{< eventlist >}}

Reply:
What is the difference between passed parameters and `context.Params`?
I mean why should I pass the params if I can access them anyway through `Params`?

Reply:
In eventlist, you have to use `.Get` to access the params that we use for example here in _index.md:

`{{< eventlist type="range" date="next_12m" >}}`

From L10 of eventlist.html, we could access all Params of the .md files as `.Params.homepage`. However, in L64 we call the partial `event(.html)` which we have to give a context. Normally, this would be `.`, so we could access `.Params.homepage` as before.

BUT, we have to pass some variables as well, and we cannot merge them in the current context `.`. That is why with the `dict` function we "rename" `.` to the parameter `.context`, so we can still access the homepage via `.context.Params.homepage`. The variable `$day` is accessible as `.day` in event.html